app/graphql/types/customer/customer.gql

```diff
@@ -31,4 +31,12 @@ type Customer {
   id: ID!
   info: CustomerInfo
   authenticationQuestions: CustomerAuthenticationQuestions
+  businesses: [CustomerBusiness]
+}
+
+type CustomerBusiness {
+  customerId: String!
```

Comment:
Is this the same id returned in the customer as id?

Reply:
If so probably not needed

Reply:
To resolve the `CustomerBusiness.roles` and `CustomerBusiness.privileges` fields, their resolvers need both a `customerId` and `businessId` to call the auth endpoint and match the person: https://github.com/DEFRA/ffc-customer-registry-api/pull/37/files#diff-25387beb0e1dd02e49dab2d4dcbe28c853f629d9307769131e844cb9c6233c1cR15

So the value needs to be there to be passed down the resolver chain, but yeah, from a schema point of view, it's a duplicate. It doesn't _have_ to be in the schema I guess. If I remove it, it still works.